### PR TITLE
BUG Fix missing return and incorrect phpdoc

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

File: code/extensions/Lumberjack.php [hunk 1]
```diff
@@ -57,9 +57,10 @@ public function updateCMSFields(FieldList $fields) {
 	/**
 	 * Augments (@link Hierarchy::stageChildren()}
 	 *
-	 * @param $staged DataList
-	 * @param $showAll boolean
-	 **/
+	 * @param showAll Inlcude all of the elements, even those not shown in the menus.
```
Comment (assertchris) [inline code/extensions/Lumberjack.php]: *Include

**Comment by assertchris** [comment]
"fixing"

File: code/extensions/Lumberjack.php (hunk 1 above)
Comment (tractorcow) [inline code/extensions/Lumberjack.php]: Doh

**Comment by tractorcow** [comment]
fixed :)

**Pull request merged**